product_docs/docs/pgd/5/manual/08-using-pgd-cli.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Step 8 - Using PGD CLI
+navTitle: Using PGD CLI
+deepToC: true
+---
+
+## Using PGD CLI
+
+The PGD CLI command is authenticated in the same way as the psql command. Use an account which you have previously configured authentication for used to run it.
```

Comment:
something isn't parsing in this sentence:  "Use an account which you have previously configured authentication for used to run it."  something about the "for used to" part....